jython_cli.java

```diff
@@ -0,0 +1,210 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
+
+//DEPS dev.jbang:jash:0.0.3
+//DEPS org.tomlj:tomlj:1.1.1
+//DEPS org.python:jython-standalone:2.7.4
+//JAVA 21
+
+import java.io.*;
+import java.nio.file.*;
+import java.util.*;
+import java.util.stream.Stream;
+import dev.jbang.jash.Jash;
+import org.tomlj.Toml;
+import org.tomlj.TomlParseResult;
+import org.tomlj.TomlTable;
+
+import org.python.util.jython;
+
+public class jython_cli {
+
+    // Increment this number with each new release of python-cli
```

Comment:
You mean "release of jython-cli", right?

Reply:
Correct. Fixed